src/sql/src/plan/statement/show.rs

```diff
@@ -456,8 +456,10 @@ pub fn show_indexes<'a>(
 ) -> Result<ShowSelect<'a>, PlanError> {
     let mut query_filter = Vec::new();
 
-    if on_object.is_none() && from_schema.is_none() {
+    if on_object.is_none() && from_schema.is_none() && in_cluster.is_none() {
         query_filter.push("on_id NOT LIKE 's%'".into());
+        let schema_spec = scx.resolve_active_schema().map(|spec| spec.clone())?;
```

Comment:
The docs https://materialize.com/docs/sql/show-indexes/ now seem to be outdated:

> Defaults to public in the current database.

Reply:
Updated.